docker-compose.yaml

```diff
@@ -46,3 +46,11 @@ services:
       - './pipeline/trigger.py'
       - '--settings=/home/kernelci/config/kernelci.conf'
       - 'run'
+
+  kcidb:
+    <<: *base-service
+    container_name: 'kcidb'
+    command:
+      - './pipeline/kcidb.py'
+      - '--settings=/home/kernelci/config/kernelci.conf'
+      - 'run'
```

Comment:
Please call `/usr/bin/env python3` explicitly, see this comment with a similar issue: https://github.com/kernelci/kernelci-pipeline/pull/14#issuecomment-1036390726

Also please add a missing line return at the end of the file to remove the git warning.